### ISSUE TITLE
New Powerup indicators break replay rendering/playback

### ISSUE THREAD

**Issue by catalyst518** [chrahunt/TagProReplays#257]
The new powerup indicators break replays.

You need to update this file: https://github.com/chrahunt/TagProReplays/blob/master/src/js/modules/tiles.js with the new index values used by the pup indicators: 6.12, 6.22, 6.32, and 6.42. They use the same tile coordinates as their respective powerup.

**Comment by zmunro** [comment]
@catalyst518 did something take the place of whatever was at index 6.11? I switched the previews to be: 
```
  6.12: { tile: "previewjukejuice",    coordinates: { x: 12, y: 4 }, tileSize: 40, img: "tiles", preview: true, emptyCoordinates: { x: 12, y: 8 } },
  6.22: { tile: "previewrollingbomb",  coordinates: { x: 12, y: 5 }, tileSize: 40, img: "tiles", preview: true, emptyCoordinates: { x: 12, y: 8 } },
  6.32: { tile: "previewtagpro",       coordinates: { x: 12, y: 6 }, tileSize: 40, img: "tiles", preview: true, emptyCoordinates: { x: 12, y: 8 } },
  6.42: { tile: "previewspeed",        coordinates: { x: 12, y: 7 }, tileSize: 40, img: "tiles", preview: true, emptyCoordinates: { x: 12, y: 8 } },
  ```
  and now I am seeing an error like this:
  ```
  tpr:renderer:error Error locating floor tile description for 6.11
  ```

**Comment by zmunro** [comment]
Made a pull request for this issue
[https://github.com/chrahunt/TagProReplays/pull/258](https://github.com/chrahunt/TagProReplays/pull/258)

**Comment by chrahunt** [comment]
The basic issue was fixed in #268 by gracefully handling unknown tiles. This will be released in [1.6.5](https://github.com/chrahunt/TagProReplays/milestone/4) along with updates to the powerup tiles and a temporary fix for team portals.
